src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CaseActionsForCaseWorkerControllerTest.java

```diff
@@ -466,6 +467,19 @@ public void amendHearing() throws Exception {
                 .andExpect(jsonPath("$.warnings", nullValue()));
     }
 
+    @Test
```

Comment:
What about testing for other HTTP responses?

Reply:
added tests for them as well